gateway/1.9/modules/ROOT/pages/flex-gateway-version-lifecycle.adoc

```diff
@@ -0,0 +1,63 @@
+= Flex Gateway Version Lifecycle
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+== Edge and LTS Release Channels
+
+Flex Gateway releases are available in two release channels, Edge and Long-term Support (LTS). 
+
+[%header,cols="4*a"]
+|===
+
+| Channel Type | Release Cadence | Differentiators | Support Coverage
+
+| *Edge* 
+| Three times per year 
+| Includes new features more often, and has a shorter support period for each release.
+a| 
+* 4 months of Standard Support
+* 4 months of Extended Support
+
+| *LTS*
+| Once per year
+| Bundles new features in an annual release, and has a longer support period for each release.
+a|
+* 12 months of Standard Support
+* 12 months of Extended Support
+
+|===
+
+Consider the following factors to determine which release channel is the best choice for your requirements.
+
+* Edge release channel
++ 
+Mulesoft releases Flex Gateway Edge versions three times per year. Use the Edge channel if you want to stay on top of the latest features and feel comfortable with a more frequent release cycle and auto-upgrade cycle. The Edge channel includes the latest features and innovations available on the MuleSoft platform.
+
+* LTS release channel
++ 
+MuleSoft releases Flex Gateway Long-Term Support (LTS) versions once a year. Use the LTS channel if you want to avoid upgrading to a new version frequently. The LTS channel adopts new capabilities after they have been introduced via Edge releases.
```

Comment:
```suggestion
MuleSoft releases Flex Gateway Long-Term Support (LTS) versions once a year. Use the LTS channel if you want to avoid upgrading to a new version frequently. The LTS channel adopts new capabilities after they've been introduced via Edge releases.
```